aten/src/ATen/native/native_functions.yaml

```diff
@@ -7129,11 +7129,11 @@
 - func: choose_qparams_optimized(Tensor input, int numel, int n_bins, float ratio, int bit_width) -> (Tensor, Tensor)
   variants: function
 
-- func: _autocast_to_reduced_precision(Tensor(a) self, bool cuda_enabled, bool cpu_enabled, ScalarType cuda_dtype, ScalarType cpu_dtype) -> Tensor(a)
+- func: _autocast_to_reduced_precision(Tensor(a) self, bool cuda_enabled, bool cpu_enabled, bool xla_enabled, ScalarType cuda_dtype, ScalarType cpu_dtype, ScalarType xla_dtype) -> Tensor(a)
```

Comment:
I think this is bc-breaking? not very familiar with how to do this, but I think we'd need to add upgraders for jit, right?